src/headers/tomcrypt_pk.h

```diff
@@ -251,7 +251,9 @@ typedef enum ecc_signature_type_ {
    /* raw R, S values */
    LTC_ECCSIG_RFC7518    = 0x1,
    /* raw R, S, V (+27) values */
-   LTC_ECCSIG_ETH27      = 0x2
+   LTC_ECCSIG_ETH27      = 0x2,
+   /* SSH + ECDSA signature format defined by RFC5656 */
+   LTC_ECCSIG_RFC5656    = 0x3
```

Comment:
I'm not 100% sure if this shouldn't be within `#ifdef LTC_SSH`?

if we're already at it, can you please directly add a `,` at the end :)

Reply:
I think it makes sense to _define_ the signature format in all cases, whether or not the implementation is built into the library. If `LTC_SSH` isn't defined, then it'll fall through to the "unknown format" errors in the functions.

I think a trailing comma in an enum results in a compiler warning. Not sure, though?

Reply:
> I think it makes sense to _define_ the signature format in all cases

agreed!



> I think a trailing comma in an enum results in a compiler warning. Not sure, though?

shouldn't on modern compilers :) (e.g. the `ltc_asn1_XX` enums have it as well)

Reply:
> shouldn't on modern compilers

Yup. I thought I'd seen this somewhere recently, but apparently not - it compiles cleanly.